pyanaconda/ui/tui/spokes/installation_progress.py

```diff
@@ -131,14 +131,22 @@ def show_all(self):
 
         util.ipmi_report(IPMI_FINISHED)
 
+        if self.instclass.eula_path is not None:
```

Comment:
Please, remove `is not None`, so it is consistent with `pyanaconda/ui/gui/hubs/progress.py`.

Reply:
Fixed.